assets/src/edit-story/components/tabview/index.js

```diff
@@ -57,21 +59,28 @@ const Tab = styled.li.attrs(({ isActive }) => ({
   cursor: pointer;
   border: none;
   background: none;
-  color: ${({ theme }) => theme.colors.fg.tertiary};
-  font-family: ${({ theme }) => theme.DEPRECATED_THEME.fonts.tab.family};
-  font-size: ${({ theme }) => theme.DEPRECATED_THEME.fonts.tab.size};
-  font-weight: ${({ theme }) => theme.DEPRECATED_THEME.fonts.tab.weight};
   padding: 10px 0px;
-  margin: 0 20px;
+  margin: 0 16px;
```

Comment:
Hm, 16px is be correct by design, however, since the svgs in the library tab use `margin: 0 -4px` then it should still be `margin: 0 20px` for the library tabs to get the correct margin:
<img width="92" alt="Screenshot 2021-03-02 at 09 08 18" src="https://user-images.githubusercontent.com/3294597/109611638-d3f9af00-7b2e-11eb-801d-baddb06753f0.png">

Reply:
But not for the text labels in the inspector, though... I'll figure out how to solve this.

Reply:
I think I managed to address it all. This I dealt with [like so](https://github.com/google/web-stories-wp/pull/6570/files#diff-6156709620006f9663d475867b53e1691d28446fa4636001541b2248eb98bbe8R49) - which I actually think is the right approach here.